DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateCourses/Index.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateCourses
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model DelegateCoursesViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/courseSetup.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Delegate courses";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter">
+    <nav class="side-nav-menu" aria-label="Side navigation bar">
+      <partial name="~/Views/TrackingSystem/Delegates/Shared/_DelegatesSideNavMenu.cshtml" model="DelegatePage.DelegateCourses" />
+    </nav>
+  </div>
+
+  <div class="nhsuk-grid-column-three-quarters">
```

Comment:
I've also just merged 597 which adds a loading spinner to a bunch of pages where we use JS search/sort/paginate. This will page will also need the loading spinner added to it once you get the latest changes from master into the branch. The loading spinner is mentioned in the docs: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/19577569424/Javascript+search+sort+paginate+functionality+with+js-search . I'm happy to help give pointers on its implementation as well.